src/manage-apps/using-eddy.md

```diff
@@ -0,0 +1,35 @@
+# Eddy
+
+The Pop!\_Shop does not include all applications available for installation in Pop!\_OS. Eddy is an alternative GUI application that can handle installation of deb packages downloaded from the internet.
+
+---
+
+## Installing Deb Packages with Eddy
+
+Eddy is an [open-source](https://github.com/donadigo/eddy) deb package installer that comes pre-packaged with Pop!\_OS.
```

Comment:
Should "Eddy" be the link in this section, instead of "open-source"?

The link, to me, looked like it would take me to a description of Open Source, and not to the source code of Eddy itself.

Reply:
- Linked "Eddy" instead of "open source"
- changed "pre-packaged" to "pre-installed'
- updated with correct link to our fork